### PR TITLE
D2.0p3.6

### PR DESCRIPTION
**Pull request by lakshaychopra**
Updated Dockerfile and docker-compose.yml files.
settings.py file is updates to run in dockerised environment.
Added missing requirements in requirements.txt file.

### CODE REVIEW

File: src/librehatti/settings.py [hunk 1]
```diff
@@ -20,15 +20,15 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.mysql",
-        "NAME": "librehatti3",
-        "USER": "root",
-        "PASSWORD": "J@1diamond",
-        "HOST": "localhost",
-        "PORT": "",
+        "NAME": "librehatti",
```
Comment (jasvir99) [inline src/librehatti/settings.py:23]: Please use env file to define such information. Create file for dev, qa and prod

**Review by jasvir99 — CHANGES_REQUESTED** [review]

**Comment by jasvir99** [comment]
@lakshaychopra Please rebase your branch.

**Comment by lakshaychopra** [comment]
On Wed, May 13, 2020 at 1:57 AM Jasvir Singh Grewal <
notifications@github.com> wrote:

> @lakshaychopra <https://github.com/lakshaychopra> Please rebase your
> branch.
>
Okay


-- 
Lakshay Chopra
Blog- codedynamo.wordpress.com

**Comment by lakshaychopra** [comment]
> @lakshaychopra Please rebase your branch.

Getting conflicts in rebasing the branch.
Should I skip those commits showing conflicts.

**Comment by jasvir99** [comment]
@lakshaychopra  You have to fix the conflicts

**Comment by lakshaychopra** [comment]
> @lakshaychopra You have to fix the conflicts

@jasvir99 i have rebased my master branch.

**Comment by jasvir99** [comment]
@lakshaychopra You have to fix pre-commits checks also. Please refer https://github.com/GreatDevelopers/LibreHatti#raising-pull-request

**Comment by jasvir99** [comment]
Also, I don't see the description and title updated.

**Comment by jasvir99** [comment]
I'll update it by myself. Thanks for your help. I am closing this PR.

**Pull request closed without merging**